package.json

```diff
@@ -35,6 +40,7 @@
     "url": "https://github.com/Pomax/node-flickrapi.git"
   },
   "scripts": {
-    "test": "node test"
+    "test": "node test",
+    "testWP": "cd browser && webpack --config --display-error-details webpack.config.js"
```

Comment:
`cd` is a bit fragile, it's safer to call `webpack --config --display-error-details --config ./browser/webpack.config.js` instead.